README.md

```diff
@@ -87,6 +89,7 @@
 * [Coingate](https://coingate.com) - Accept payment in Dai and 50 other crypto currencies.
 * [Dai Card](https://daicard.io/) - Instant, off-chain, peer to peer, Dai transactions using browser-based UI.
 * [Living Room of Satoshi](https://www.livingroomofsatoshi.com/) - Pay with confidence at Australia's largest crypto payments gateway.=
+* [Request](https://app.request.network) - Create cryptocurrency invoices and request payments in Dai, ETH and other ERC20 tokens.
```

Comment:
This seems like a cool service-- something I might use, honestly! I wonder if this one also might be better off in "Payments." What do you think?

Reply:
This one or Gilded? :) The Request (links to app.) one should already be placed under Payments, agree.

Now we're busy - there's also the API which might be of interest to add later but I wasn't sure  if it would be a good addition to the list. It's essentially a distribution channel for devs to integrate the functionalities of Request (create decentralized payment requests, i.e. invoices) into their existing products without needing to know the ins & outs of blockchain development. I've made a dedicated landing page for it here, check it out and would love to hear what you think: https://request.network/en/developers/request-api/

Reply:
I think payments would be the better fit

Reply:
Agree - good for me 👍